### ISSUE TITLE
Long Range Scanner Issues

### ISSUE THREAD

**Issue by Sly9sky** [Eclipse-Station/NEV-Northern-Light#1629]
Two issues noted with long range scanner during play:

-Long Range Scanner receives no external power. This is because it needs a cable to run up to its core, it will not draw power from an APC

-Disassembling and reassembling it breaks it. Reassembly creates the Long Range Scanner core but not any of the periphery arms. During disassembly, no additional parts are created from the periphery "arms" if that's what was supposed to happen. As a result, a reassembled Long Range Scanner will not work.

**Comment by Sly9sky** [comment]
To clarify, this central part of the Scanner is the only part that is built on reassembly of the Long Range Scanner
![97dee675699ebc54eebde1cf91a460bc](https://user-images.githubusercontent.com/87826427/159191567-662ce4ef-0bb6-417f-8709-e6ef6bb6024e.png)
.

**Comment by Valanthe500** [comment]
Testing on my end, I can confirm the scanner does indeed have a power cable, I'm unable to tell when this was changed however.

I've also tested deconstructing the Long Range Scanner. I disassembled the device all the way down to a machine frame. Upon rebuilding the scanner, it gave me an unsecured long range scanner. Once I secured it (apply wrench), I was able to right click the scanner and deploy the conduit.
